src/components/paragraphs/sup-carousel/sup-carousel-paragraph.tsx

```diff
@@ -65,71 +65,75 @@ const Slide = ({slideParagraph, isTopHero}: { slideParagraph: ParagraphSupCarous
   return (
     <article
       aria-labelledby={slideParagraph.id}
-      className={twMerge("relative centered-container text-white", clsx({
+      className={twMerge("relative centered-container text-white w-full h-full min-h-full lg:min-h-screen", clsx({
         "text-center": !leftImage,
         "text-center lg:text-left": leftImage,
         "lg:pt-[300px] lg:mb-[-300px]": isTopHero
       }))}
     >
-
+      <figure className="absolute top-0 left-0 w-full h-full overflow-hidden">
+        <Image
+            className="relative object-cover"
+            src={bgImage.url}
+            alt=""
+            fill
+            sizes="100vw"
+          />
+      </figure>
       <div
-        className={clsx("-z-10 absolute h-full w-full top-0 left-0", {
+        className={clsx("absolute top-0 left-0 block h-full w-full bg-opacity-80", {
```

Comment:
personally, i think the `mix-blend-multiply` looks better. it actually interacts with the image to produce the result. What are thoughts from design?

Reply:
Discussed and this will be g2g for now.